code/modules/mob/living/carbon/human/human.dm

```diff
@@ -268,8 +274,17 @@
 					to_chat(usr, "Added by [c.author] at [c.time]")
 					to_chat(usr, "----------")
 				to_chat(usr, "<b>Notes:</b> [R.fields["notes"]]")
+				to_chat(usr, "<b>Security Record:</b> [SEC.fields["past_records"]]") //SKYRAT EDIT ADDITION BEGIN - EXAMINE RECORDS
 				return
 
+			if(href_list["genrecords"])
+				if(!H.canUseHUD())
+					return
+				if(!HAS_TRAIT(H, TRAIT_SECURITY_HUD))
+					return
+				var/datum/data/record/GEN = find_record("name", perpname, GLOB.data_core.general)
+				to_chat(usr, "<b>General Record:</b> [GEN.fields["past_records"]]") //SKYRAT EDIT END
```

Comment:
You should already have access to the general record, now this would be better.
```suggestion
			 //SKYRAT EDIT ADDITION BEGIN - EXAMINE RECORDS
			if(href_list["genrecords"])
				if(!H.canUseHUD())
					return
				if(!HAS_TRAIT(H, TRAIT_SECURITY_HUD))
					return
				to_chat(usr, "<b>General Record:</b> [general_record.fields["past_records"]]")
			//SKYRAT EDIT END
```